### PR TITLE
Add support for corner and shadow on UICollectionViewCell

### PR DESCRIPTION
**Pull request by mmadjer**
Add support for corner and shadow on UICollectionViewCell.

It would be good if someone could have a look at this, cheers.

### CODE REVIEW

**Review by tbaranes — APPROVED** [review]
Looks good! 🎉

File: IBAnimatable/CornerDesignable.swift
Comment (JakeLin) [inline IBAnimatable/CornerDesignable.swift]: Sorry, I started to use two space a tab because most of the time, I use Mac book only (with 15-inch screen) on the train or on the couch 😂, That gives me more space to read. Can we stick with this rule for this project?

File: IBAnimatable/CornerDesignable.swift
Comment (JakeLin) [inline IBAnimatable/CornerDesignable.swift]: May use a `let layerName = "cornerSideLayer" ` to avoid duplicated Strings.

**Review by JakeLin — APPROVED** [review]
Just a few of small comments, I leave it to you to address them or merge it, thanks.

File: IBAnimatable/CornerDesignable.swift
Comment (mmadjer) [reply IBAnimatable/CornerDesignable.swift]: Yep, good point. Will apply.

File: IBAnimatable/CornerDesignable.swift
Comment (mmadjer) [reply IBAnimatable/CornerDesignable.swift]: Sorry, for sure. I haven't pay attention to the default tabs. Will change it back.

**Comment by tbaranes** [comment]
@mmadjer You can merge this you are done with it! No need to wait once you got an approval and fixed the feedbacks (if there's some) 💯

**Comment by JakeLin** [comment]
@mmadjer thanks 🎉🎉🎉

**Pull request merged**
